docs/development/ide-setup/intellij-setup.md

```diff
@@ -1,23 +1,27 @@
 ## *IntelliJ* Setup
 
-### Plugins:
-  - *Google Java Format*
-    - *settings > Other settings > google-java-format*
-    - Check 'enable'
+Setting up *IntelliJ* is pretty simple.  There are 4 plugins to install with the associated settings, and some settings to download and import.  
```

Comment:
nit, avoiding dependencies in documentation makes it easier to maintain. Specifically the "4 plugins", if we add a plugin to the list either that will be out of date or we'll need to update it. One could say instead "There are a few plugins to install..." which would make it more generic and less likely to become out of date.

Reply:
Will keep that in mind.  Thanks